### PR TITLE
Add scutech cms driver

### PR DESCRIPTION
**Pull request by weibangtuo**
**What this PR does / why we need it**:

HostBase Replication Driver for Scutech Cloud Migration System(CMS), support create, enable,disable,delete interface.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

None.

**Special notes for your reviewer**:

None.

**Release note**:

```release-note
Driver for Scutech Cloud Migration System.
```

### CODE REVIEW

File: contrib/drivers/scutech/cms/replication.go [hunk 1]
```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019 Scutech Co., Ltd. All Rights Reserved.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (wisererik) [inline contrib/drivers/scutech/cms/replication.go]: Change the copyright to **`Copyright 2019 The OpenSDS Authors.`**

File: contrib/drivers/scutech/cms/scmstask.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019 Scutech Co., Ltd. All Rights Reserved.
+//
```
Comment (wisererik) [inline contrib/drivers/scutech/cms/scmstask.go]: same here

File: contrib/drivers/scutech/cms/scmsutils.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2019 Scutech Co., Ltd. All Rights Reserved.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (wisererik) [inline contrib/drivers/scutech/cms/scmsutils.go]: same here

File: contrib/drivers/scutech/cms/scmstask.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+func checkVolume(c *CmsTask, volumeId string) bool {
+	_, find := c.volumeList[volumeId]
+
+	return find
+}
+
+func (t *CmsTask) AddVolume(source CmsVolume, target CmsVolume) error {
+	if findSource := checkVolume(t, source.VolumeId); findSource {
+		return fmt.Errorf("Source volume[%s] already exists", source.VolumeId)
+	}
```
Comment (wisererik) [inline contrib/drivers/scutech/cms/scmstask.go]: Source should be source.(it should start with lower case in error log)

File: contrib/drivers/scutech/cms/scmstask.go [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+	if findTarget := checkVolume(t, target.VolumeId); findTarget {
+		return fmt.Errorf("Target volume[%s] already exists", target.VolumeId)
+	}
```
Comment (wisererik) [inline contrib/drivers/scutech/cms/scmstask.go]: same here

**Comment by xing-yang** [comment]
Please add release note as this is a new feature.

File: contrib/drivers/scutech/cms/replication.go [hunk 6]
```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019 The OpenSDS Authors.
+//
```
Comment (leonwanghui) [inline contrib/drivers/scutech/cms/replication.go]: The license style is not right.

**Pull request merged**